codegenerator/cli/templates/static/codegen/src/ink/components/CustomHooks.res

```diff
@@ -0,0 +1,76 @@
+module InitApi = {
+  type body = {
+    @as("version") envioVersion: string,
+    @as("apiToken") hasApiToken: bool,
+    usesHyperSync: bool,
+    config: ConfigYAML.t,
```

Comment:
@DZakh note I aded "usesHyperSync" field so no extra parsing is needed on the server side yet. I'm still sending across the reduced ConfigYAML (redacted addresses and rpc secrets etc) to allow us to parse info if we need for messaging later on.